sanic_httpauth.py

```diff
@@ -173,25 +179,34 @@ def __init__(self, scheme=None, realm=None, use_ha1_pw=False):
         self.generate_opaque_callback = None
         self.verify_opaque_callback = None
 
-        def _generate_random():
-            return md5(str(self.random.random()).encode("utf-8")).hexdigest()
-
         def default_generate_nonce(request):
-            request.ctx.session["auth_nonce"] = _generate_random()
-            return request.ctx.session["auth_nonce"]
+            self.nonce = self._generate_random()
+            if use_session:
```

Comment:
I don't see `use_session` defined anywhere, or am I missing something?

Reply:
nevermind, I need to drink some coffee

Reply:
@MihaiBalint you have right, there is an error. @KostSvitlana pay ATTN